src/modules/account/views/MyFolders.tsx

```diff
@@ -663,14 +663,18 @@ export const AccountMyFolders: FC<DefaultSeoInfo & AccountMyFolders> = ({ url, f
 
 	return (
 		<VisitorLayout>
-			{renderOgTags(
-				tText('pages/account/mijn-mappen/folder-slug/index___mijn-mappen') +
-					` | ${activeFolder?.name || folderSlug}`,
-				tText(
+			<SeoTags
+				title={
+					tText('pages/account/mijn-mappen/folder-slug/index___mijn-mappen') +
+					` | ${activeFolder?.name || folderSlug}`
+				}
+				description={tText(
 					'pages/account/mijn-mappen/folder-slug/index___mijn-mappen-meta-omschrijving'
-				),
-				url
-			)}
+				)}
+				imgUrl={undefined}
+				translatedPages={[]}

```

Comment:
Same here, `translatedPages` defaults to `[]`

Reply:
was already fixed in a later PR